tests/solvers/test_radau5.py

```diff
@@ -26,6 +26,36 @@
 from assimulo.exception import *
 from assimulo.lib.radau_core import Radau_Exception, Radau_Common
 import scipy.sparse as sp
+import numpy as np
+import re
+
```

Comment:
Can we have some documentation of this class.

Reply:
Added docstring in new commit:

    """Auxiliary class for creating problems (both explicit and implicit) 
    that simulate a Keyboardinterrupt (Ctrl + c) in the rhs f or jacobian.
    
    Set 'fcn' or 'jac' to True to enable KeyboardInterrupt Exceptions in f or jac."""

Reply:
good